test/MUXControls.Test/Infra/Application.cs

```diff
@@ -246,7 +246,7 @@ private UIObject LaunchApp(string packageName)
                     Log.Comment("Launch successful!");
                     break;
                 }
-                catch (UIObjectNotFoundException)
+                catch (Exception)
                 {
                     Log.Comment("UAPApp.Launch failed to find app, dumping UIA tree...");
```

Comment:
Error statement still implies that this is a "not found" error, but now it could be just about anything. Log out the exception message instead?